reframe/utility/__init__.py

```diff
@@ -81,7 +81,9 @@ def import_module_from_file(filename, force=False):
         filename = os.path.join(filename, '__init__.py')
 
     # Express filename relative to reframe
-    rel_filename = os.path.relpath(filename, sys.path[0])
+    rfm_root = os.path.realpath(os.path.join(os.path.dirname(__file__),
+                                '../..'))
```

Comment:
Isn't that equivalent to this?

```suggestion
    rfm_root = os.path.realpath(reframe.INSTALL_PREFIX)
```

And another question is that it's not obvious to me the purpose of `os.path.realpath` here.

Reply:
The `os.path.realpath` was to get the resulting path without the `../../`